lib/awful/tooltip.lua

```diff
@@ -546,6 +551,22 @@ function tooltip:set_margins_topbottom(val)
     self:set_margin_topbottom(val)
 end
 
+--- Set the margins between the tooltip and its parent
+--
+-- @DOC_awful_tooltip_gaps_EXAMPLE@
+--
+-- @property tooltip_gaps
```

Comment:
The property is called `gaps`. `set_gaps` will make `my_tooltip.gaps = 42` work. The fact that `self._private.tooltip_gaps` exists rather than `self._private.gaps` will have some little side effects, but since this is undocumented, I don't mind much.